### PR TITLE
Add support for NTP on cell

### PR DESCRIPTION
**Pull request by duduita**
- https://github.com/WildernessLabs/Meadow_Issues/issues/714
- https://github.com/WildernessLabs/Meadow.Core/pull/560

### CODE REVIEW

File: Source/implementations/f7/Meadow.F7/Devices/F7CellNetworkAdapter.cs
Comment (ctacke) [inline Source/implementations/f7/Meadow.F7/Devices/F7CellNetworkAdapter.cs:174]: Has the OS already changed the current time here (so if I call DateTime.Now in an event handler, will the time be correct)?

Comment (duduita) [reply Source/implementations/f7/Meadow.F7/Devices/F7CellNetworkAdapter.cs:174]: There's an open [PR](https://github.com/WildernessLabs/Meadow/pull/636) in the Meadow.OS that aims to support NTP on cell/ethernet. 

So, I think we can wait for the OS PR to be merged before merging this PR. But, let me know if you want any changes.

**Review by ctacke — APPROVED** [review]

**Pull request merged**
